src/app/dto/CardsDTO/CreateCardDTO.ts

```diff
@@ -5,12 +5,14 @@ export class CreateCardDTO {
   question: string;
   answer: string;
   rating: string;
```

Comment:
Why rating is string?

Reply:
Didn't think about it. I'll change to number.